bundle.Dockerfile

```diff
@@ -15,7 +15,7 @@ LABEL com.redhat.delivery.operator.bundle=true
 
 # This second label tells the pipeline which versions of OpenShift the operator supports.
 # This is used to control which index images should include this operator.
-LABEL com.redhat.openshift.versions="=v4.17"
+LABEL com.redhat.openshift.versions="=v4.18"
```

Comment:
Has CVP started supporting 4.18? This may fail the CVP tests if not.

Reply:
Hmm CVP plans to support 4.18 by end of Sept. I can drop this for now, and create a task to followup and change to this

Also just curious, what benefit do the CVP tests provide us -- are they only useful for the final release build? Would failing CVP tests prevent us from turning builds over to QE?

Reply:
I looked this up and found it's not only the test failure but the index images do not get generated due to this.